code/sorting/src/cocktail_sort/cocktail_sort.cpp

```diff
@@ -0,0 +1,116 @@
+//Returns Sorted elements after performing Cocktail Selection Sort
+//It is a Sorting algorithm which chooses the minimum and maximum element in an array simultaneously, 
+//and swaps it with the lowest and highest available position iteratively or recursively
+
+
+#include<iostream>
+using namespace std;
+
+//Iterative Version
+
+void CocktailSelectionSort(vector <int> &vec,int low,int high)
```

Comment:
Please model your code after the related sorting files, such as [this one](https://github.com/OpenGenus/cosmos/blob/master/code/sorting/src/quick_sort/quick_sort.cpp). It should serve as a good example as to what idiomatic code is expected for C++.